cpp/src/slice2swift/Gen.cpp

```diff
@@ -267,9 +265,7 @@ Gen::TypesVisitor::visitClassDefStart(const ClassDefPtr& p)
         out << eb;
     }
 
-    //
-    // For each Value class we generate an extension method in ClassResolver
-    //
+    // For each Value class we generate an extension method in ClassResolver.
```

Comment:
I would update the comment to stress this name is derived from the type ID, not the mapped name.

Reply:
If we have no test for this feature (unmarshaling a remapped class), we need to add one.

Reply:
I added the requested comment, and opened an issue.
This isn't specific to Swift, we need to add/augment these tests in all languages.